### PR TITLE
add ALEMAL (AS4242421525) peering in Stockholm STO1 

### PR DESCRIPTION
**Pull request by AlexanderMalmstrom**
Hi!

I have had some issues with my node and saw that it had been offline long enough for you to understandably remove our peering.

### CODE REVIEW

File: routers/router.sto1.yml [hunk 1]
```diff
@@ -1 +1,12 @@
...
+  asn: 4242421525
+  ipv6: fe80::1525
+  multiprotocol: true
+  extended_nexthop: true
+  sessions: [ipv6]
+  wireguard:
+    remote_address: se1.alemal.se
+    remote_port: 50207
+    public_key: Bg91m2vr46p8oqCqB4j5JQwRoYyb/rwR3iy5X0hany4=
+
```
Comment (jvoss) [inline routers/router.sto1.yml:12]:
```suggestion
```
Remove this line to pass the YAML linter.

**Review by jvoss — CHANGES_REQUESTED** [review]
No problem! Happy to have you back as a peer!

Just a minor nit from the Linter.

**Comment by AlexanderMalmstrom** [comment]
> No problem! Happy to have you back as a peer!
> 
> Just a minor nit from the Linter.

Should be fixed now

**Comment by jvoss** [comment]
Looks good to me! This should be deployed in the next few minutes.

**Review by jvoss — APPROVED** [review]

**Review by yzguy — APPROVED** [review]

**Pull request merged**